docker-compose.yaml

```diff
@@ -1,13 +1,43 @@
 version: "3.4"
 services:
   authz:
-    image: authz
     container_name: authz-service
-    build: /home/authz/git
+    build:
+      context: .
+      dockerfile: Dockerfile.authz
     ports:
-      - 80:5000
+      - "80:5000"
     tty: true
+    networks:
+      app-network:
+        ipv4_address: 172.22.0.6
+
+  nginx:
+    image: nginx:latest
+    container_name: nginx
+    build:
+      context: .
+      dockerfile: Dockerfile.nginx
+    depends_on:
+      - authz
+    networks:
+      app-network:
+        ipv4_address: 172.22.0.5
     volumes:
-      - Docker_Authz:/home/authz/app
+      - nginx_log:/var/log/nginx
+
+networks:
+  app-network:
+    ipam:
+      driver: default
+      config:
+        - subnet: 172.22.0.0/16
+          gateway: 172.22.0.1
+
 volumes:
-  Docker_Authz:
\ No newline at end of file
+  nginx_log:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
+      device: /home/authz/nginx_logs
```

Comment:
I as one of developers of authz service want to have the ability to run docker-compose and test our application locally. With current configuration of `device` its not possible because i dont have home/authz folder. So we need to think of a different solution here